scripts/check_frontend_coverage.py

```diff
@@ -284,8 +284,9 @@
     'practice-tab.directive.ts',
     'pretest-question-backend-api.service.ts',
     'preview-tab.directive.ts',
-    'profile-link-image.directive.ts',
-    'profile-link-text.directive.ts',
+    'profile-link-image-backend-api.service.ts',
```

Comment:
Do not introduce new files here. You need to cover any new files you add with tests.

Reply:
Okay, @seanlip, I'll make tests for it!

Reply:
Sounds good, thanks.

Also, next time, please make sure to read the documentation before you modify stuff. There is already a comment above this list that explains explicitly what not to do.